content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Add a job to testgrid"
+date: 2020-10-28T11:14:39-04:00
+draft: false
+---
+
+## What is testgrid?
+Testgrid is an aggregator for job results across many runs, allowing us to display test results over time and thus giving an indication of current health, especially in the face of flaky tests.
+
+## What are release-informing/blocking/broken jobs?
+* `Blocking jobs` are those that signal widespread failure of the platform. These are traditionally the core end-to-end test runs on our major platforms and upgrades from previous versions. 
+* `Informing jobs` are a broader suite that test the variety of enviroments and configurations our customers expect. 
+* `Broken jobs` are those that have a known, triaged failure that prevents their function for a sustained period of time (more than a week).
+
+## How to decide if a job should qualify as a release-informing job?
+Release informing jobs in the [release controller config](https://github.com/openshift/release/tree/master/core-services/release-controller/_releases) get run every time we build a new payload for that release.
+So you need to ask yourself whether the given job:
+1) is reliable enough to provide a useful signal when it fails
+2) covers an important function that determines whether the payload is viable or not, which is not covered by other informing or blocking jobs
+
+If it does not meet the qualifications for being a release controller informing job, it can still be added to testgrid (and thus [sippy](https://sippy.ci.openshift.org/)) by adding it to the [_allow-list](https://github.com/openshift/release/blob/master/core-services/testgrid-config-generator/_allow-list.yaml). In which case it won't be run against every new payload that is built, it would instead just run on whatever periodic schedule you configured it for.
```

Comment:
This is unrelated to how a job is put into the release blocking or informing bucket, move

Reply:
This is explaining the factors behind whether the job is added to the testgrid via the release controller or whether its just added as an entry in the allow-list. I think the user should probably be able to understand from the doc why they would want to add the job in the allow-list instead of directly in the RC

Reply:
Right, but it's not related to this section. I'm not opposed to having the information it just needs to be laid out succinctly with appropriate topics